Tangem/Modules/Main/ActionButtons/Coordinator/ActionButtonsCoordinatorProtocol.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ActionButtonsCoordinatorProtocol.swift
+//  Tangem
+//
+//  Created by GuitarKitty on 24.10.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+protocol ActionButtonsCoordinatorProtocol {
```

Comment:
```suggestion
protocol ActionButtonsRoutable / ActionButtonsRouter {
```
суффикс `Protocol` не используем)